src/Cake.Core/IO/Arguments/SecretArgument.cs

```diff
@@ -35,5 +35,14 @@ public string RenderSafe()
         {
             return "[REDACTED]";
         }
+
+        /// <summary>
+        /// Returns a <see cref="string" /> that represents the current object.
+        /// </summary>
+        /// <returns>A string that represents the current object.</returns>
+        public override string ToString()
+        {
+            return this.Render();
```

Comment:
* Should call `RenderSafe` to avoid rendering secret stuff by mistake.
* Remove `this` qualifier.